core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -255,12 +255,12 @@ object KafkaConfig {
   val LogFlushIntervalMsProp = "log.flush.interval.ms"
   val LogFlushOffsetCheckpointIntervalMsProp = "log.flush.offset.checkpoint.interval.ms"
   val LogPreAllocateProp = "log.preallocate"
-  val MessageFormatVersionProp = "message.format.version"
+  val LogMessageFormatVersionProp = "log.message.format.version"
```

Comment:
Yeah, unfortunately we can't apply this now as a matter of rule.
I mean for the new configuration - can we use "log."+ LogConfig.MessageFormatVersionProp instead of "log.message.format.version".

Not a big deal, but seems like a simple improvement?